client/flutter/lib/components/page_scaffold/back_arrow.dart

```diff
@@ -27,10 +31,14 @@ final _paint = Paint()..color = const Color(0xFF646C77);
 /// Draws an arrow.
 class BackArrow extends StatelessWidget {
   @override
-  Widget build(BuildContext context) => IconButton(
-        // TODO make sure this is localised properly
-        tooltip: MaterialLocalizations.of(context).backButtonTooltip,
-        icon: PathWidget(path: _icon, paint: _paint),
-        onPressed: () => Navigator.pop(context),
-      );
+  Widget build(BuildContext context) => Transform.rotate(
+    // back arrow need to point right in the RTL languages.
+    angle: Constants.isDirectionRTL(context) ? 180 * pi/180 : 0,
```

Comment:
Ideally we would mirror rather than rotate.

Reply:
Yup, in this case, we need another SVG data for the right arrow.

Reply:
You can transform around the Y axis using `Transform(transform :Matrix4.rotationY(180)...)`
see https://stackoverflow.com/questions/58047009/flutter-how-to-flip-an-icon-to-get-mirror-effect

Reply:
I will give it a try, but I think it's the same behavior.